automation/upload_test_artifacts.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -ex
+
+SCP_SRV="grisge.info"
+SCP_BASEDIR="upload"
+SCP_USERNAME="nmstate"
+SCP_KEY="./automation/upload_test_artifacts.key"
+BASE_URL="http://grisge.info/nmstate"
```

Comment:
Just wondering, can you setup letsencrypt for this? Not a blocker, though.

Reply:
Done. Through I need create a cron job for auto-renew.